src/test/kotlin/com/atlassian/performance/tools/lib/table/GenericPlainTextTable.kt

```diff
@@ -0,0 +1,102 @@
+package com.atlassian.performance.tools.lib.table
+
+import java.util.*
+
+/**
+ * Provides generic functionality for creating basic ascii tables
+ *
+ * Basic usage
+ *
+ *  val table = GenericPlainTextTable()
+ *  table.addRow(listOf( "", "Run", "Apdex", "Total Actions", "Error %", "Satisfactory %", "Tolerable %"))
+ *  table.addRow(listOf( "best", "5", "0.7713201820940819", "23155", "0.389", "59.490", "34.684"))
+ *  table.addRow(listOf( "worst", "6", "0.7677944046844503", "23143", "0.380", "58.925", "35.125"))
+ *  println(table.generate())
+ *
+ * giving
+ *
+ *  *------*----*-------------------*--------------*--------*---------------*------------*
+ *  |      | Run| Apdex             | Total Actions| Error %| Satisfactory %| Tolerable %|
+ *  *------*----*-------------------*--------------*--------*---------------*------------*
+ *  | best | 5  | 0.7713201820940819| 23155        | 0.389  | 59.490        | 34.684     |
+ *  | worst| 6  | 0.7677944046844503| 23143        | 0.380  | 58.925        | 35.125     |
+ *  *------*----*-------------------*--------------*--------*---------------*------------*
+ *
+ *
+ * Assumes the first row added is the header, but that can be defined in the constructor. Dynamically works out column widths so you don't have to.
+ */
+class GenericPlainTextTable (private val headerRowCount: Int = 1) {
+    private val _builder = StringBuilder()
```

Comment:
No need for `_`, because the `private` keyword does the job

Reply:
I'll edit it

Reply:
It doesn't directly go against the [Kotlin Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules)